### DISCUSSION TITLE
Eslint plugin:@nrwl/nx/angular should not overwrite certain rules like @typescript-eslint/member-ordering

### DISCUSSION THREAD

**Discussion by stefan-schweiger** [nrwl/nx#5181] [Feature Requests]
Currently `plugin:@nrwl/nx/angular` sets certain rules like `@typescript-eslint/member-ordering`. As far as I can tell this is also already set in `plugin:@nrwl/nx/typescript` in the root `/.eslintrc.json`.

But with `plugin:@nrwl/nx/angular` being extended by default in all angular libs you can't overwrite the member-ordering yourself in the root configuration but would need to set it in each `<lib>/.eslintrc.json`.

I'm certain there are some eslint configurations which make sense to set on a lib level, but those which are already set by `plugin:@nrwl/nx/typescript` shouldn't be included in them.

Are there any arguments against evaluating and excluding certain eslint rules?

**Comment by markonyango** [comment]
I absolutely agree with this point. It took me quite some time to find out why some rules were not "taking" and others were. The only solution I found so far was to create a separate .eslintrc-base.json file which holds the rules object. That way I can extend it in each apps and libraries .eslintrc.json file (must be the last import because order of extending files matters):
```
"extends": [
        "plugin:@nrwl/nx/angular",
        "plugin:@angular-eslint/template/process-inline-templates",
        "../../.eslintrc-base.json"
      ],
```
The setup as it is being created by the CLI is currently counter-intuitive.

**Reply by stefan-schweiger** [reply]
I've already filed an issue for this https://github.com/nrwl/nx/issues/5259

@JamesHenry I still think this was wrongly closed. Any comment on that?

**Comment by JamesHenry** [comment]
I don't think this needs to be both a discussion and an issue, so is it possible to close this one as there is more context already on the issue thread?

**Reply by stefan-schweiger** [reply]
I would close it, but I don't see how :/  Am I overlooking something?

**Reply by JamesHenry** [reply]
Oh interesting, maybe Discussions can't be closed?
